README.md

```diff
@@ -56,7 +54,11 @@ Run local App Engine dev server:
 Configure your local development parameters by copying the `dev_config/config.yml.dist` file to `dev_config/config.yml`.  Edit `config.yml` as needed.  This file is .gitignore'd.  See comments in the `config.yml.dist` file for more details.
 
 #### MySQL database configuration
-Create a new MySQL for this application.  Configure the MySQL connection in `config.yml`.  Then, run `bin/console doctine:migrations:migrate` to build your local database.
+Create a new MySQL for this application.  Configure the MySQL connection in `config.yml`. 
+
+Then, run `bin/console doctine:migrations:migrate` to build your local database.
+
+Then, run `docker exec -it healthpro-web bin/console doctine:migrations:migrate` to build your database in docker container.
```

Comment:
We have the `bin/dx` shortcut that I usually use. You could include that here, and maybe also an explanation of what it is.

```suggestion
Then, run `bin/dx bin/console doctrine:migrations:migrate` to execute the necessary Doctrine migrations.

(`bin/dx` is a convenience wrapper for executing a command inside the Docker web container. You can also run `bin/dx bash` to open a shell into the container.)
```